apps/openchallenges/app/src/_app-theme.scss

```diff
@@ -1,15 +1,13 @@
 @use 'sass:map';
 @use '@angular/material' as mat;
-@use 'libs/openchallenges/themes/src/fonts' as fonts;
```

Comment:
Note that the path does no longer include the prefix `libs/` thanks to specifying `stylePreprocessorOptions` in `project.json`.